### PR TITLE
Create GitHub footer and add myself to the project

### PR DESCRIPTION
**Pull request by PatoGuereque**
This pull request will greatly boost search engine results by adding an additional person to the project and adding a footer with a GitHub repository link. Furthermore, the delicate choice of colors for both Dean and myself passes the AAA standards for large text, resulting in a better experience for the website visitors and aiming for an accessible experience. Lastly, people who are curious about looking at the source code of the website will be able to easily do so by browsing the repository linked instead of relying on trying to find it on GitHub. 

Extensive testing was conducted to ensure these changes are compatible with Cloudflare Pages. Finally, each commit was carefully crafted to follow the project's coding style and guarantee a uniform codebase.

### CODE REVIEW

**Comment by deanveloper** [comment]
LGTM :shipit:

File: pages/index.tsx [hunk 1]
```diff
@@ -50,6 +50,15 @@ const Subtitle = styled.h3`
...
 `;
 
+const Footer = styled.p`
+	margin-top: 25px;
+`;
+
+const FooterLink = styled.a`
+	color: #ffffff;
+	text-decoration: underline;
+`;
```
Comment (deanveloper) [inline pages/index.tsx:60]: probably need to add styling for `:visited`, i'll fix tho

Comment (deanveloper) [reply pages/index.tsx:60]: (eventually)

Comment (PatoGuereque) [reply pages/index.tsx:60]: eventually™

Comment (deanveloper) [reply pages/index.tsx:60]: we actually didn't need it pepela

**Pull request merged**